src/V3Options.cpp

```diff
@@ -917,639 +1108,544 @@ void V3Options::parseOptsList(FileLine* fl, const string& optdir, int argc, char
+    DECL_OPTION("-lint-only", OnOff, &m_lintOnly);
+    DECL_OPTION("-main", OnOff, &m_main);  // Undocumented future
+    DECL_OPTION("-no-pins64", CbCall, [this]() { m_pinsBv = 33; });
+    DECL_OPTION("-order-clock-delay", OnOff, &m_orderClockDly);
+    DECL_OPTION("-pvalue+", CbPartialMatch,
+                [this](const char* varp) { addParameter(varp, false); });
+    DECL_OPTION("-pins64", CbCall, [this]() { m_pinsBv = 65; });
+    DECL_OPTION("-pins-sc-uint", CbOnOff, [this](bool flag) {
+        m_pinsScUint = flag;
+        if (!m_pinsScBigUint) m_pinsBv = 65;
+    });
+    DECL_OPTION("-pins-sc-biguint", CbOnOff, [this](bool flag) {
+        m_pinsScBigUint = flag;
+        m_pinsBv = 513;
+    });
+    DECL_OPTION("-pins-uint8", OnOff, &m_pinsUint8);
+    DECL_OPTION("-pp-comments", OnOff, &m_ppComments);
+    DECL_OPTION("-private", CbCall, [this]() { m_public = false; });
+    DECL_OPTION("-prof-cfuncs", OnOff, &m_profCFuncs);
+    DECL_OPTION("-profile-cfuncs", OnOff, &m_profCFuncs);  // Undocumented, renamed
+    DECL_OPTION("-prof-threads", OnOff, &m_profThreads);
+    DECL_OPTION("-protect-ids", OnOff, &m_protectIds);
+    DECL_OPTION("-public", OnOff, &m_public);
+    DECL_OPTION("-public-flat-rw", CbOnOff, [this](bool flag) {
+        m_publicFlatRW = flag;
+        v3Global.dpi(true);
+    });
+    DECL_OPTION("-quiet-exit", OnOff, &m_quietExit);
+    DECL_OPTION("-relative-cfuncs", OnOff, &m_relativeCFuncs);
+    DECL_OPTION("-relative-includes", OnOff, &m_relativeIncludes);
+    DECL_OPTION("-report-unoptflat", OnOff, &m_reportUnoptflat);
+    DECL_OPTION("-savable", OnOff, &m_savable);
+    DECL_OPTION("-sc", CbCall, [this]() {
+        m_outFormatOk = true;
+        m_systemC = true;
+    });
+    DECL_OPTION("-skip-identical", OnOff, &m_skipIdentical);
+    DECL_OPTION("-stats", OnOff, &m_stats);
+    DECL_OPTION("-stats-vars", CbOnOff, [this](bool flag) {
+        m_statsVars = flag;
+        m_stats |= flag;
+    });
+    DECL_OPTION("-structs-unpacked", OnOff, &m_structsPacked);
+    DECL_OPTION("-sv", CbCall, [this]() { m_defaultLanguage = V3LangCode::L1800_2017; });
+    DECL_OPTION("-threads-coarsen", OnOff, &m_threadsCoarsen);  // Undocumented, debug
+    DECL_OPTION("-trace", OnOff, &m_trace);
+    DECL_OPTION("-trace-coverage", OnOff, &m_traceCoverage);
+    DECL_OPTION("-trace-params", OnOff, &m_traceParams);
+    DECL_OPTION("-trace-structs", OnOff, &m_traceStructs);
+    DECL_OPTION("-trace-underscore", OnOff, &m_traceUnderscore);
+    DECL_OPTION("-underline-zero", OnOff, &m_underlineZero);  // Deprecated
+    DECL_OPTION("-verilate", OnOff, &m_verilate);
+    DECL_OPTION("-vpi", OnOff, &m_vpi);
+    DECL_OPTION("-Wpedantic", OnOff, &m_pedantic);
+    DECL_OPTION("-x-initial-edge", OnOff, &m_xInitialEdge);
+    DECL_OPTION("-xml-only", OnOff, &m_xmlOnly);
+
+    DECL_OPTION("-CFLAGS", CbVal, {this, &V3Options::addCFlags});
+    DECL_OPTION("-comp-limit-blocks", Set, &m_compLimitBlocks);  // Undocumented
```

Comment:
Perhaps now that we have cleaner parsing we should insertion sort these into above by option name?

Reply:
Sorted in db4bc964f46379600b89b6fcb4a94b86c830d4dd